src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,10 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
-    public String getRandomColor() {
-        return null;
+    public String getRandomColor(Random random) {
+        int index = random.nextInt(Color.values().length);
+        return Color.values()[index].toString();
```

Comment:
Please, do not use `toString()` here.
See [common mistakes](https://mate-academy.github.io/jv-program-common-mistakes/java-core/oop/lottery#use-name-for-getting-string-representation-of-enum-constants)